src/icon-manifest/supportedFolders.ts

```diff
@@ -206,7 +206,7 @@ export const extensions: IFolderCollection = {
     { icon: 'ravendb', extensions: ['ravendb'], format: FileFormat.svg },
     {
       icon: 'route',
-      extensions: ['route', 'routes', '_route', '_routes'],
+      extensions: ['route', 'routes', '_route', '_routes', 'routers'],
```

Comment:
What about this? Can you provide a reference to a renowned project using this? I cannot see why someone would use a `routers` folder. Is it because you use more than one `router implementation` in your client applications?

Reply:
Main use case for me is for backend [express](https://expressjs.com)-powered applications when you often have a lot of [routers](https://expressjs.com/en/4x/api.html#router) that you attach to your main express app.

Reply:
I see, I was only taking into account client apps and didn't thought about `express`. You totally have a valid point there. 😉